### PR TITLE
Fixed inconsistancy issues in cse-collector

### PR DESCRIPTION
**Pull request by savitaashture**
1. data in dashboard disappears sometime.
2. Data of each instance is same.

### CODE REVIEW

File: csemonitor.go [hunk 1]
```diff
@@ -104,6 +116,25 @@ func (reporter *Reporter) Run() {
 	}
 }
 
+func (reporter *Reporter) getServiceID() {
```
Comment (tianxiaoliang) [inline csemonitor.go]: don not interact with sc for twice just get service id directly

Comment (savitaashture) [reply csemonitor.go]: The interaction will happen only when service ID deletes...
But as per your comment if we change this way we need to interact with SC every time even if service ID is correct.

Comment (tianxiaoliang) [reply csemonitor.go]: you can put this logic only in go-chassis, not in this repo

Comment (savitaashture) [reply csemonitor.go]: Done

File: csemonitor.go [hunk 2]
```diff
@@ -104,6 +99,18 @@ func (reporter *Reporter) Run() {
...
+	var serviceID string
+	for {
+		serviceID, _ = registry.DefaultServiceDiscoveryService.GetMicroServiceID(reporter.app,
+			reporter.service, reporter.version, reporter.environment)
+		if serviceID != "" {
+			break
+		}
+	}
+	reporter.serviceID = serviceID
```
Comment (tianxiaoliang) [inline csemonitor.go]: just set config.SelfServiceID here, don't query sc

**Pull request merged**